### PR TITLE
Add jest recommended rules

### PR DESCRIPTION
**Pull request by jeanregisser**
This will catch jest disabled (.skip,xit,etc) or focused (.only,fit,etc) that were unintentionally committed.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,42 +1,53 @@
 module.exports = {
   root: true,
   extends: [
-    'eslint:recommended',
-    'plugin:@typescript-eslint/eslint-recommended',
-    'plugin:@typescript-eslint/recommended',
+    "eslint:recommended",
```
Comment (silasbw) [inline index.js:4]: why the change to double quote?  doesn't that conflict with our linter (or prettier) rules?

Comment (jeanregisser) [reply index.js]: It used the default prettier config, I've updated it to use our own.

**Review by silasbw — APPROVED** [review]
